common/religion/religions/shamanism_group.txt

```diff
@@ -619,5 +619,68 @@
 			#Magic
 			doctrine = doctrine_life_magic_accepted
 		}
+
+		#Trogg religion
+		trogg_religion = {
+			#Main Group
+			color = rgb { 82 58 40 }
+			icon = wczu_trogg_religion
+			reformed_icon = wczu_trogg_religion
+
+			doctrine = unreformed_faith_doctrine
+
+			holy_site = ulduar #Where the Trogg were created
+			holy_site = red_desert # Uldaman, where many Trogg were held in stasis before the dwarves dug them out
+
+			#Not sure about these ones
+			holy_site = ironforge #Because the trogg want to kill the dwarves
+			holy_site = gnomeregan  #Because the trogg were disturbed by the Gnomes and took the city
+			holy_site = alterac_valley #Trogg infest a mine here
+
+			#Main Group
+			doctrine = doctrine_no_head
+
+			doctrine = tenet_natural_primitivism
+			doctrine = tenet_sanctity_of_nature
+			doctrine = tenet_human_sacrifice
+
+			#Magic
+			#Ignore all except some shaman, they don't know any other magic
+			doctrine = doctrine_shadow_magic_ignored
+			doctrine = doctrine_disorder_magic_ignored
+			doctrine = doctrine_death_magic_ignored
+			doctrine = doctrine_elemental_spirit_magic_accepted
+			doctrine = doctrine_elemental_decay_magic_ignored
+			doctrine = doctrine_elemental_fire_magic_ignored
+			doctrine = doctrine_elemental_water_magic_ignored
+			doctrine = doctrine_elemental_air_magic_ignored
+			doctrine = doctrine_elemental_earth_magic_accepted
+		}
+
+		#Drogbar religion
+		drogbar_religion = {
+			#Main Group
+			color = rgb { 117 72 39 }
+			icon = wczu_drogbar_religion
+			reformed_icon = wczu_drogbar_religion
+
+			doctrine = unreformed_faith_doctrine
+			
+			holy_site = icemist #These are placeholders, can't put proper ones until Broken Isles are mapped
+			holy_site = taunkale
+			holy_site = wintergarde
+			holy_site = njord
+			holy_site = grizzlemaw
+
+			#Main Group
+			doctrine = doctrine_no_head
+
+			doctrine = tenet_natural_primitivism
+			doctrine = tenet_sanctity_of_nature
+			doctrine = tenet_ancestor_worship
+
+			#Magic
+			doctrine = doctrine_elemental_earth_magic_approved #Drogbar have strong control over earth
```

Comment:
I would say that the drogbar wouldn't mind the Decay.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/115989703-aaaa4c00-a5d0-11eb-959f-a2ea840bcdeb.png)

</details>